src/priceClient/adapters/acrossApi.ts

```diff
@@ -2,12 +2,17 @@ import { msToS, PriceFeedAdapter, TokenPrice } from "../priceClient";
 import { BaseHTTPAdapter } from "./baseAdapter";
 
 type AcrossPrice = { price: number };
+type AcrossApiArgs = {
+  name?: string;
+  host?: string;
+  timeout?: number;
+};
 const defaultTimeout = 5000; // mS
 
 export class PriceFeed extends BaseHTTPAdapter implements PriceFeedAdapter {
-  constructor(name: string, { host, timeout }: { host?: string; timeout?: number }) {
+  constructor(args?: AcrossApiArgs) {
```

Comment:
nit: I don't think it needs to be this complicated. I think you can keep the exact same code as before with a default arg of `{}`. Thoughts? Or am I missing something?

```suggestion
  constructor(args: AcrossApiArgs = {}) {
```